app/views/users/show.html.erb

```diff
@@ -57,6 +35,36 @@
       <% end %>
       </div>
 
+      <% unless !user_signed_in? && !@user.community_user.privilege?('unrestricted') %>
+      <% if @user.valid_websites_for.size.positive? %>
+      <div>
+        <p><strong>Extra fields</strong></p>
+        <table class="table is-with-hover">
+          <% @user.valid_websites_for.each do |w| %>
+          <tr>
+            <td><%= w.label %></td>
+            <td>
+	      <% if w.url[0,4] == 'http' %>
+	        <%= link_to w.url, w.url, rel: 'nofollow' %>
+	      <% else %>
+	      <%= w.url %>
+	      <% end %>
+	    </td>
```

Comment:
A few more tab stops slipped past here

Reply:
Fixed those new ones to speed things up (on a tangent, we seem to have 9 other files with tab stops)